src/Javis.jl

```diff
@@ -119,6 +119,37 @@ function projection(p::Point, l::Line)
     return c * v + o
 end
 
+"""
+    preprocess_frames!(objects::Vector{<:AbstractObject})
+
+Computes the frames for each object and action based on the user defined frames that the
+user can provide like `Rel` and `:same`.
```

Comment:
Could you add to the docstring about why this preprocess is needed for the `get_javis_frame` method? I understand that it defines CURRENT_OBJECT, but I had to do a bit of poking around to figure out what exactly it was doing.

Reply:
I've added a docstring. `CURRENT_OBJECT` is not that important I think. It can maybe get removed there at some point.